### PR TITLE
Basic run

### PR DESCRIPTION
**Pull request by richfitz**
This PR implements a basic run endpoint:

* `POST /nimue/run` with payload like the example in `tests/testthat/nimue-run-example.json` (or see the schema `NimueRun.schema.json`)

Also

* `GET /countries` which returns a list of known countries

The `TEST` country is `GBR` stripped down to the first 40 days of the epidemic and runs for me in about 0.1s so will be nicer to test against.

The testing is extremely light - this is basically a lightly tweaked version of the example code provided by OJ, wrapped up in porcelain. I expect there are many combinations of inputs that we might protect against but currently nothing like that is done :)

Quick start:

```
docker run --rm -d -p 8321:8321 mrcide/cometr:basic-run
curl --silent -X POST -H "Content-Type: application/json" -d @cometr/tests/testthat/nimue-run-example.json http://localhost:8321/nimue/run | jq
```

### CODE REVIEW

**Review by EmmaLRussell — COMMENTED** [review]
Don't have much to contribute on the R code, but I'll have a go at joining this up with comet next. Would be great to get some test fvps and cumulativePercentPopVacced values in.

File: inst/schema/Countries.schema.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "NimueRunPars",
```
Comment (r-ash) [inline inst/schema/Countries.schema.json]:
```suggestion
    "id": "Countries",
```

**Review by r-ash — APPROVED** [review]
Looks good to me. One typo in schema mentioned.

You've probably thought about this already but if your model run takes a while for some countries then it will block other requests being handled (like we saw with calibration in hintr). If the test cases 0.1s probably not a big issue, but not sure how bigger countries will fare? Also probably not a big deal if you aren't expecting too many users.

There are a couple of TODOs sitting around too - assuming you're happy leaving them around?

**Comment by richfitz** [comment]
> You've probably thought about this already but if your model run takes a while for some countries then it will block other requests being handled (like we saw with calibration in hintr). If the test cases 0.1s probably not a big issue, but not sure how bigger countries will fare? Also probably not a big deal if you aren't expecting too many users.

Yes, this will take a bit long in practice - up to a second in real life. We'll either move to async with workers or (given this is stateless) put a load balancer in front of a pool of api clients

**Comment by richfitz** [comment]
And yes, both TODO's are pending confirmation from the science team

**Pull request merged**